instant.ts

```diff
@@ -0,0 +1,299 @@
+  const envVars = []
+  for(let envVar in packageInfo.metadata.environmentVariables) {
+    envVars.push({"Environment Variable": envVar, "Default Value": packageInfo.metadata.environmentVariables[envVar], "Updated Value": env[envVar]})
+  }
+  console.table(envVars)
+}
+
+// Main script execution
+const main = async () => {
+  const allPackages = getInstantOHIEPackages()
+  console.log(
+    `Found ${Object.keys(allPackages).length} packages: ${Object.values(
+      allPackages
+    )
+      .map((p) => p.metadata.id)
+      .join(', ')}`
+  )
+
+  const main = commandLineArgs(
+    [
+      {
+        name: 'command',
+        defaultOption: true
+      }
+    ],
+    {
+      stopAtFirstUnknown: true
+    }
+  )
+
+  let argv = main._unknown || []
+
+  // main commands
+  if (['init', 'up', 'down', 'destroy'].includes(main.command)) {
+    const mainOptions = commandLineArgs(
+      [
+        {
+          name: 'target',
+          alias: 't',
+          defaultValue: 'docker'
+        },
+        {
+          name: 'only',
+          alias: 'o',
+          type: Boolean
+        },
+        {
+          name: 'dev',
+          alias: 'd',
+          type: Boolean
+        }
+      ],
+      { argv, stopAtFirstUnknown: true }
+    )
+
+    console.log(`Target environment is: ${mainOptions.target}`)
+
+    argv = mainOptions._unknown || []
+    let chosenPackageIds = argv
+
+    if (
+      !chosenPackageIds.every((id) => Object.keys(allPackages).includes(id))
+    ) {
+      throw new Error(`Deploy - Unknown package id in list: ${chosenPackageIds}`)
+    }
+
+    if (chosenPackageIds.length < 1) {
+      chosenPackageIds = Object.keys(allPackages)
+    }
+
+    if (!mainOptions.only) {
+      // Order the packages such that the dependencies are instantiated first
+      chosenPackageIds = orderPackageIds(allPackages, chosenPackageIds)
+    }
+
+    if (mainOptions.dev) {
+      mainOptions.mode = 'dev'
+    } else {
+      mainOptions.mode = 'prod'
+    }
+
+    if (['destroy', 'down'].includes(main.command)) {
+      chosenPackageIds.reverse()
+    }
+
+    console.log(
+      `Selected package IDs to operate on: ${chosenPackageIds.join(', ')}`
+    )
+
+    switch (mainOptions.target) {
+      case 'docker':
+        for (const id of chosenPackageIds) {
+          logPackageDetails(allPackages[id])
+          await runBashScript(`${allPackages[id].path}docker/`, 'compose.sh', [
+            main.command
+          ])
+        }
+        break
+      case 'k8s':
+      case 'kubernetes':
+        for (const id of chosenPackageIds) {
+          await runBashScript(
+            `${allPackages[id].path}kubernetes/main/`,
+            'k8s.sh',
+            [main.command]
+          )
+        }
+        break
+      case 'swarm':
+        for (const id of chosenPackageIds) {
+          await runBashScript(`${allPackages[id].path}/`, 'swarm.sh', [
```

Comment:
Could you please add this package detail logger to the other two deploy options 🙃 
This was my change on instant that needed to be removed when I did the weird merge.
Basically gives a nice sanity check as components are launching

```suggestion
        for (const id of chosenPackageIds) {
          logPackageDetails(allPackages[id])
          await runBashScript(
            `${allPackages[id].path}kubernetes/main/`,
            'k8s.sh',
            [main.command]
          )
        }
        break
      case 'swarm':
        for (const id of chosenPackageIds) {
          logPackageDetails(allPackages[id])
          await runBashScript(`${allPackages[id].path}/`, 'swarm.sh', [
```

Reply:
done